server.js

```diff
@@ -63,8 +63,8 @@
 const http = require('http');

```

Comment:
Consider using a descriptive variable name instead of 'PORT'. This can improve code readability and maintainability.